src/tests/testRock.cc

```diff
@@ -40,6 +40,9 @@
  * test the store framework
  */
 
+/// Define Chrooted variable to use Instance.o
+bool Chrooted = false;
```

Comment:
> I also added the global variable Chrooted to the tests to make them use Instance.o.

@yadij, there are several documented problems and a TODO in main.cc that suggest removing problematic chroot support from Squid. Would you support such a removal (in a dedicated PR, of course)?

Reply:
I can't answer that at this time, nor an a generic way. IIRC there was some controversial opinions on what to change vs remove, and also our downstream have some chroot related needs that complicate the situation.